src/knot_protocol.c

```diff
@@ -123,7 +125,8 @@ static const struct schema {
 	{KNOT_VALUE_TYPE_FLOAT, UNIT_LONG_MASK		},
 	{KNOT_VALUE_TYPE_INT, UNIT_SPEED_MASK		},
 	{KNOT_VALUE_TYPE_FLOAT, UNIT_VOLFLOW_MASK	},
-	{KNOT_VALUE_TYPE_INT, UNIT_ENERGY_MASK		}
+	{KNOT_VALUE_TYPE_INT, UNIT_ENERGY_MASK		},
+	{KNOT_VALUE_TYPE_INT, UNIT_ACCELERATION_MASK	}
```

Comment:
Is there a reason to have it as INT rather than FLOAT? IMO this would cause a quite significant gaps in possible values.
Starting at V0 = 0, with a = 1m/s² after 10 seconds V = 10 m/s (36 km/h)
Starting at V0 = 0, with a = 1.6m/s² after 10 seconds V = 16 m/s (~58 km/h) --> This would have to be rounded to either 1 or 2, but is not really near any IMO.
Starting at V0 = 0, with a = 2m/s² after 10 seconds V = 20 m/s (72 km/h)

Not to consider that we could not have anything lower than 1 m/s.

Reply:
If I did not miss anything in my math, but it's just to show the point.

Reply:
I Agree with Paulo.